source/features/closing-remarks.tsx

```diff
@@ -80,19 +92,21 @@ async function addReleaseBanner(text = 'Now you can release this change'): Promi
 		return;
 	}
 
-	const url = canCreateRelease() ? (
-		isRefinedGitHubRepo()
-			? 'https://github.com/refined-github/refined-github/actions/workflows/release.yml'
-			: buildRepoURL('releases/new')
-	) : undefined;
+	const url = createReleaseUrl();
+	const bannerContent = {
+		icon: <TagIcon className="m-0"/>,
+		classes: ['rgh-bg-none'],
+		text,
+	};
+
 	attachElement('#issue-comment-box', {
 		before: () => (
 			<TimelineItem>
 				{createBanner(url ? {
-					text,
+					...bannerContent,
 					action: url,
-					buttonLabel: <><TagIcon/> Draft a new release</>,
-				} : {text})}
+					buttonLabel: 'Draft a new release',
+				} : bannerContent)}
```

Comment:
`RequireAllOrNone` types make optional properties annoying to use. Feel free to try locally